workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -256,19 +256,21 @@ dds.list <- map(lst, lcdbwf::make.dds, salmon.files=NULL, remove.version=TRUE,
 ```{r results, dependson='dds_list', cache=TRUE}
 # Docs: https://lcdb.github.io/lcdb-wf/rnaseq-rmd.html#results
 
-res.list <- list()
-
 # NOTE: Example contrast #1----------------------------------------------------
 #   Using the example data, this compares treatment group to control group.
 #   Change to reflect your experiment.
 # IMPORTANT: Control must always be last
-res.list[['main']] <- list(
+#   Change the letter prefix of the contrast to adjust order of appearance in the html
+#   Each contrast will appear in the res.list without the prefix "contr_[A-z]_".
+#   Make a separate chunk for each contrast.
+contr_a_main <- list(
     res=lfcShrink(dds.list[['main']], contrast=c('group', 'treatment', 'control'), type='normal'),
     dds='main',
     label='Using a log2FoldChange threshold of 0'
 )
+```
 
-
+```{r results2, dependson='dds_list', cache=TRUE}
```

Comment:
the naming of every chunk will probably break the tests that ensure each named chunk has a corresponding entry in the docs. We could get around this by having these examples have a prefix (I guess `results`) and that prefix would be skipped in that test. Or remove the names from these chunks.